packages/v4/src/content/design-guidelines/usage-and-behavior/card-view/cards.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Cards
+section: usage and behavior
+hideTOC: true
+showTitle: true
+---
+
+A **card** is a square or rectangular container of related information. Cards symbolize units of information, and each one acts as an entry point for users to access more details. For example, in dashboards and catalog views, cards function as a preview of a detailed page.  
+
+A card usually consists of  3 parts: 
+
+<img src="./img/card.png"  alt="3 parts of a card"  width="394"/>  
+
+
+1. **Header**: Title of the information group
+2. **Body**: Brief details about the group
+3. **Footer**: Links to access more information about the group 
+
+Cards are flexible, and you can customize them in a variety of ways:
+- A card can contain a header with an image, actions, and a body.
+- A card can contain only a header and a body. 
+- A card can contain a header with an image, a body, and footer. 
+
+## Card types 
+PatternFly supports two types of cards: 
+1. **Compact (recommended)**: Compact cards use 16px spacing with an 8px spacer between the header and body and 16px between the body and footer.
+2. **Large**: Large cards use 24px spacing with a 16px spacer between the header and body and 24px between the body and footer.  
+
+<img src="./img/card-types.png"  alt="compact and standard card types"  width="779"/> 
```

Comment:
Found a small inconsistency here. In the text it's referred to as "Large" but in the image I referred to it as "Standard" card. @mcarrano which one is the correct one. If it's large let me know so I can update the image! :)

Reply:
Good question. I think we settled on "Large." Can you confirm @gdoyle1 ? But I think it sounds wrong to say "Large card title because the size of the card title is not different. Can we update to just call this a "Card title" in either case?

Reply:
It seems like in pf org documentation it's called "Basic". Also, I can definitely update the card titles to just say "Card title"

Reply:
Sorry for the delayed response @mcarrano @mmenestr @abigaeljamie - Margot and I settled on:
1. **Compact (recommended)**: Compact cards use 16px spacing with an 8px spacer between the header and body and 16px between the body and footer.
2. **Standard**: Standard cards use 24px spacing with a 16px spacer between the header and body and 24px between the body and footer.  

We spoke with Matt Nolting and found that in code, you will get the "standard" spacing by default and only will receive the "compact" spacing if you apply the` pf-m-compact`. modifier. So I think that makes the most sense! I will update in sketch library too.